scripts/asm-installer/install_asm

```diff
@@ -496,7 +502,11 @@ OPTIONS:
                                       omitted if not installing optional
                                       features. To add multiple files, specify
                                       them with multiple options one at a time.
-
+  --ca_name|--ca-name    <CA NAME>    Required only if --ca option is private_ca.
+                                      Name of the ca in the GCP CAS service used to
+                                      sign certificates in the format
```

Comment:
I think `sign certificates. The format should be` is clearer, since it removes the possibility that someone reads this as "the format that GCP CAS signs certificates with is 'projects/...'"